### ISSUE TITLE
Not independent?

### ISSUE THREAD

**Issue by jenstornell** [yetanother-blog/indiepen#73]
It says it's independent. When using the code generator it's like below:

```text
<iframe class="indiepen" src="https://indiepen.tech/embed/?url=https%3A%2F%2Fexample.com%2Fasdasdsa&tab=html" style="width: 100%; overflow: hidden; display: block; border: 0;" title="Indiepen Embed" loading="lazy" width="100%" height="450"></iframe>
```

You use https://indiepen.tech. So if you suddenly don't want to maintain it anymore and remove the domain you will kill all the projects that depends on it in an instance.

Also you may add a purchase subscription because this way you really lock people in with a dependency that should not really be needed.

**Comment by HenrikFricke** [comment]
Fair point, for us it's independent because it's open-source, we share what we do and how we implement things. This approach is different to other solutions on the market.

But I agree, we have to do a better job in providing documentation for self-hosting or maybe even provide Indiepen as a web component or something like this so you don't rely on our hosting. 

So far, we developed Indiepen for our own blog and did the extra mile to provide a code snippet generator so more people can benefit from a cookieless embed. 

Feel free to host Indiepen on your own domain, fork it or do whatever you want. We are here to assist you. 

Have a nice day 🙂

**Comment by bitstarr** [comment]
Hope you share a selfhosting guide soon.
